uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -968,15 +942,21 @@ define([
       }
 
       /**
+       * Initialize user's role to either owner, manager or admin
+       */
+      function initUserRole () {
+        role = this.directiveOptions.userRole;
```

Comment:
Why do you even need this at all? Either you do it at the top of the controller (does it make sense to initialize `role` to an empty string?) or you do it in the `_.init()`